common/character_templates/dvg_agitators.txt

```diff
@@ -38,3 +38,22 @@
 		chance = 100
 	}
 }
+
+dvg_giancarlo_iannarella_template = {
+	first_name = Giancarlo
+	last_name = Iannarella
+	is_agitator = yes
+	add_character_role = politician

```

Comment:
Error: "Wrong type parsed in create_character effect: add_character_role, near line: 46" in file: "common/character_templates/dvg_agitators.txt" near line: 46

Reply:
Okay, I fixed them all by moving them into on_created = { } within the create_character = { }